.github/workflows/lint.yml

```
@@ -52,6 +52,25 @@ jobs:
       - name: pylint test
         run: bash tests/lint/pylint/run.sh .
 
+  ruff:
+    name: ruff
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v4
+
+      - name: Setup Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.8"
```

Is it possible to consistent with the required version on `Setup Python` on pylint job?

```suggestion
          python-version: "3.9"
```

---

Done